app/assets/stylesheets/_functions.scss

```diff
@@ -0,0 +1,7 @@
+@function image-src($image) {
```

Comment:
This function is required so that we can load an image via Storybook's webpack or Sprockets. The `$environment` variable is only ever set when webpack for Storybook is running.

Reply:
is this something we could benefit from in our codebase? should this be part of `app/assets/stylesheets/config/import.scss`